Examples/r/live_objects/example.h

```diff
@@ -0,0 +1,14 @@
+/* File : example.h */
+
+#include <string>
+
+class Foo
+{
+  static unsigned count;
+public:
+  Foo();
+  ~Foo();
+  static unsigned get_count();
+};
+
+static Foo* verify_no_memory_leak(const std::string& message, Foo* foo);
```

Comment:
No newline at end of file.  I'd look and see if you can configure your editor such that you don't keep doing that.